msocasecharge.yaml

```diff
@@ -0,0 +1,176 @@
+entityDefinitions:
+
+  person:
+    fqn: "nc.SubjectIdentification"
+    entitySetName: "MSOInmates"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        column: "PCP"
+    name: "person"
+ 
+  jailstay:
+    fqn: "ol.jailstay"
+    entitySetName: "MSOJailstay"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        column: "SYSID"
+    name: "jailstay"
+
+  charge:
+    fqn: "ol.offense"
+    entitySetName: "MSOCharge"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        column: "CHARGE_PK"
+      ol.eventcount:
+        type: "ol.eventcount"
+        column: "CHARGE_ORDER"
+      ol.type:
+        type: "ol.type"
+        column: "CHARGE_TYPE_ID"
+    name: "charge"
+
+  chargeevent:
+    fqn: "ol.chargeevent"
+    entitySetName: "MSOChargeevent"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        column: "CHARGE_PK"
+      ol.code:
+        type: "ol.code"
+        column: "OFFENSE_CODE"
+      ol.description:
+        type: "ol.description"
+        column: "OFFENSE_DESCRIPTION"
```

Comment:
these both actually go into ol.charge and not charge event. Please follow this page https://wiki.openlattice.com/display/DM/Charges+and+Pleas